peachjam/models/ingestors.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils import timezone
+
+from peachjam.adapters import IndigoAdapter  # noqa

```

Comment:
@longhotsummer I have imported the adapter here even though I am not using it in this file because the plugin decorator needs to run to have the adapter in the registry. Is there a better way to do this to have the adapters in the registry?